content/docs/command-reference/plot/index.md

```diff
@@ -0,0 +1,190 @@
+# plot
+
+Contains commands to visualize
+[continuous metrics](/doc/command-reference/plot#continuous-metrics) in
+structured files (JSON, CSV, or TSV): [show](/doc/command-reference/plot/show),
+[diff](/doc/command-reference/plot/diff).
+
+## Synopsis
+
+```usage
+usage: dvc plot [-h] [-q | -v] {show,diff} ...
+
+positional arguments:
+  COMMAND
+    show         Generate a plot image file from a continuous metrics file.
+    diff         Plot continuous metrics differences between commits in the DVC
+                 repository, or between the last commit and the workspace.
+```
+
+## Description
+
+DVC provides a set of commands to visualize _continuous metrics_ of machine
+learning experiments. Usual examples of plots are AUC curves, loss functions,
+and confusion matrices.
+
+Continous metrics represents a plot and should be stored as data series in one
+of the supported [file formats](#file-formats). These files are usually created
+by users or generated by user's modeling or data processing code.
```

Comment:
```suggestion
Continuous metrics represent plots, and should be stored as data series in one
of the supported [file formats](#file-formats). These files are usually created
by users or generated by user modeling or data processing code.
```

Reply:
But how are plots defined? Is there a new option in `dvc run` for these, like -m for metrics? I don't see info about this in any plot ref yet, nor an update on run itself. Does that mean these metrrics are not tracked by DVC like scalar metrics? Probably worth mentioning either way.

Reply:
@jorgeorpinel there is no plot metrics in `dvc run` yet. we should introduce one. `-p` is taken by params. What do you think about `-l`\`-L` (pLot)?

Reply:
I like `-l` (`--plots`) since it comes after `-m` (`--metrics`)

Will there also be a upper case `-L` (`--plot-no-cache`)?

All of this should probably be its own issue in the core repo 😬 I can create it if you'd like, please lmk.